cmd/fs-v1-helpers.go

```diff
@@ -162,21 +161,44 @@ func fsStatDir(statDir string) (os.FileInfo, error) {
 	return fi, nil
 }
 
-// Lookup if file exists, returns file attributes upon success
+// Is a one place function which converts all os.PathError
+// into a more FS object layer friendly form, converts
+// known errors into their typed form for top level
+// interpretation.
+func osErrToFSFileErr(err error) error {
```

Comment:
can be named as `toFSFileError()` or something similar is better i.e. it is not restricted to `os` only.

Reply:
It is because it should be used with osErrors only, toFSFileError makes it quite generic.

Reply:
I think `isSysErrNotDir()` and `isSysErrPathNotFound()` are not looking for os error.

Reply:
They are we are just looking deeper.. @balamurugana every error is os.PathError in type.